index.html

```diff
@@ -31,7 +31,7 @@
       <ul class="nav-list">
         <li class="nav-item">Portfolio</li>
         <li class="nav-item">About</li>
-        <li class="nav-item" >Contact</li>
+        <li class="nav-item">Contact</li>
```

Comment:
**Resize Text**:
Please ensure that the text, images, and other content do not overlap.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/155170951-08628115-64f9-4dfd-a346-896db389edf3.png) | ![image](https://user-images.githubusercontent.com/10332499/155171040-d2593448-f22e-4dc8-97cc-afda689367f2.png)